requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
+black==20.8b1; python_version >= '3.6'
```

Comment:
Do we really use this?

Reply:
Yes; lines 155-158 of `render_templates.py` use Black to properly format the generated Python files.